src/Mapbender/PrintBundle/Component/LayerRendererGeoJson.php

```diff
@@ -296,38 +299,59 @@ protected function drawMultiLineString($canvas, $geometry)
      */
     protected function drawPoint($canvas, $geometry)
     {
+        // promote to single-item MultiLineString and delegate
```

Comment:
the only minor point: The comment says MiltiLineString, but you're promoting to MultiPoint here